popup/popup.js

```diff
@@ -2,13 +2,13 @@ const golinkInput = document.getElementById("go-link")
 const urlInput = document.getElementById("full-link")
 const saveButton = document.getElementById("save")
 const searchQuery = document.getElementById("search")
-const mainDiv = document.getElementById("main")
-const helpDiv = document.getElementById("help")
-const helpButton = document.getElementById("help-btn")
-const homeButton = document.getElementById("home-btn")
+const mainDiv = document.getElementById("main-section")
+const helpDiv = document.getElementById("help-section")
+const helpButton = document.getElementById("btn-help")
+const homeButton = document.getElementById("btn-home")
 const itemList = document.getElementById("item-list")
 const addSection = document.getElementById("add-section");
```

Comment:
you haven't adjusted the name for this one. I assume it should be `section-expand`?